File: src/main/java/nl/hu/oop/werkboek/week1/les1/opdracht4/Student.java
package nl.hu.oop.werkboek.week1.les1.opdracht4;

public class Student {

    private String naam;

    public Student(String nm) {
        naam = nm;
    }

    public String getNaam() {
        return naam;
    }

    public String toString() {
        return "Deze student heet " + naam;
    }
}

File: src/main/csharp/HU.OOP.Week1.Opdracht4/Student.cs
namespace HU.OOP.Week1.Opdracht4;

public class Student
{
    public String Naam { get; private set; } //C# heeft een klein truukje om niet al die getXYZs te hoeven schrijven
    //Leuk, maar 'lekker belangrijk'

    public Student(String nm)
    {
        this.Naam = nm;
    }

    public override string ToString()
    {
        return "Deze student heet " + Naam;
    }
}